packages/teams-js/CHANGELOG.md

```diff
@@ -334,98 +115,106 @@ Tue, 01 Mar 2022 19:50:49 GMT
     - The following APIs have been added to the new `mail` namespace:
       - `openMailItem` is added
       - `composeMail` is added
-  - Added Chat capability and renamed `conversations` namespace to `chat`
-    - `openConversation` and `closeConversation` have been moved to `chat` namespace
-    - `getChatMembers` has been moved to `chat` namespace
-  - Added Files capability
-    - `openFilePreview` has moved from `privateAPIs` to `files` namespace
-  - Added Legacy capability
-    - The following APIs have been moved from `privateAPIs` to a new `legacy.fullTrust` namespace:
-      - `getUserJoinedTeams`
+  - Added Chat capability to new `chat` namespace to represent public chat functionality
+    - Added `openChat` and `openGroupChat` to open Teams chats with one or more users
+  - Added Conversations capability from existing `conversations` namespace to represent private chat functionality
```

Comment:
I don't understand what it means to "Add" something "from" an existing namespace. Add it to where?

Reply:
Good call out. The intent here is to explain how we took an existing namespace and made it into a capability. We added a capability, by adding an isSupported method essentially, to an existing namespace. I think this syntax is in several places. Got any ideas for how to improve it?

Reply:
oh I see. I would go with something like "Converted existing `conversations` namespace into the Conversations capability...."